### PR TITLE
make scripts executable and add shebangs

### PR DESCRIPTION
**Pull request by bmintz**
basically any file that has top-level code or
`if __name__ == '__main__'` should have a shebang line
so it can be executed as though it were a binary.

In order to do that, they must also have the execute permission bit set.

### CODE REVIEW

File: examples/image2term.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env python3
```
Comment (asciimoo) [inline examples/image2term.py]: Please use consistently `python` instead of `python3`

File: examples/flappy_birds.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
-import curses
+
+import curses-
```
Comment (asciimoo) [inline examples/flappy_birds.py]: Seems a `-` character is added accidentally, please remove it

**Comment by bmintz** [comment]
OK it should be good now

**Comment by asciimoo** [comment]
Great, thanks

**Pull request merged**